openslides/utils/views.py

```diff
@@ -88,14 +52,33 @@ def as_view(cls, *args, **kwargs):
         return ensure_csrf_cookie(view)
 
 
-class PDFView(PermissionMixin, View):
+class PDFView(View):
     """
     View to generate an PDF.
     """
-
     filename = ugettext_lazy('undefined-filename')
     top_space = 3
     document_title = None
+    required_permission = None
+
+    def check_permission(self, request, *args, **kwargs):
+        """
+        Checks if the user has the required permission.
+        """
+        if self.required_permission is None:
+            return True
```

Comment:
No this is old style code for Django views. We can do this in an extra pull request.